src/napari/_qt/widgets/qt_welcome.py

```diff
@@ -30,6 +33,14 @@ class QtVersionLabel(QLabel):
     """Label used for displaying version information."""
 
 
+# TODO: make them respect settings?
+TIPS = [
+    'You can take a screenshot and copy it to your clipboard by pressing alt+C',
+    'You can change most shortcuts from the File->Preferences->Shortcuts menu',
+    'You can right click many components of the graphical interface to access advanced controls',
+]
```

Comment:
Could we move it just below imports?  
The global constant should be before class declaration.